### ISSUE TITLE
405 Method not allowed in prod env using ocelot with eureka service discovery

### ISSUE THREAD

**Issue by AdnanInayat** [ThreeMammals/Ocelot#955]
## Steps to Reproduce the Problem

  1. Do a "POST" request

## Specifications

  - Platform: .net core 2.2 with ocelot and eureka
  - Subsystem:

**Comment by AdnanInayat** [comment]
I am using Ocelot and Eureka for my project. Everything is working perfecetly in development enviornment. By dev env I mean running everything on localhost. But when I try to run it on production, "GET" request is working perfectly but "POST" request is giving me "405 not allowed".
Any help will be appreciated :)

**Comment by fabiomalagoli** [comment]
Hi @AdnanInayat,
I've the same problem, have you found some solution? 

Tks

**Comment by Sammy210** [comment]
Hi @AdnanInayat  and @fabiomalagoli, 

I searched the entire internet for how to resolve this but to no avail. 
**PROBLEM:**
I then investigated the application console logs and noticed that despite my ocelot application (UpStream) is running on HTTPS, my "DownstreamScheme" config in ocelot.json was set to "HTTP", whereas, the destination (downstream) production application was running on HTTPS. 
**SOLUTION:**
What I did was remove HTTP to HTTPS redirection on my (downstream) production application and it solved the problem.

Despite that seems to resolve the problem (for now), I am looking forward to ensuring that everything works on HTTPS.
I will update that is done.

I hope this helps.
